policyengine_us/parameters/gov/states/id/tax/income/deductions/retirement_benefits/amount.yaml

```diff
@@ -0,0 +1,34 @@
+description: Idaho allows for this maximum retirement benefits deduction amount, based on filing status.
+
+metadata:
+  label: Idaho retirement benefit deduction amount
+  period: year
+  unit: currency-USD
+  breakdown:
+    - filing_status
+  reference:
+    - title: Individual Income Tax General Information 2022
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf#page=32
+    - title: Form 39R Resident Supplemental Schedule 2022
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EFO00088/EFO00088_12-30-2022.pdf#page=1
+    # amount not specified in legal code
+    - title: TITLE 63 REVENUE AND TAXATION CHAPTER 30 INCOME TAX 63-3022A 
+      href: https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3022a/
+    - title: Individual Income Tax Forms and Instructions 2021
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=33
+
+SINGLE:
+  2021-01-01: 37_776
+  2022-01-01: 40_140
+JOINT:
+  2021-01-01: 56_664
+  2022-01-01: 60_210
+SEPARATE:
+  2021-01-01: 37_776
+  2022-01-01: 40_140
+WIDOW:
+  2021-01-01: 37_776
+  2022-01-01: 40_140
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 37_776
+  2022-01-01: 40_140
```

Comment:
where does it specify that hoh, widow and separate are all the same as single - add reference